include/Mixer.h

```diff
@@ -151,7 +151,7 @@ class LMMS_EXPORT FxMixer : public Model, public JournallingObject
 
 	QString nodeName() const override
 	{
-		return "fxmixer";
+		return "mixer";
```

Comment:
This will break compatibility of projects and presets unless proper upgrade method is added to `DataFile::upgrade`.